libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/options/VoiceInstructionsPlayerOptions.kt

```diff
@@ -108,11 +119,12 @@ class VoiceInstructionsPlayerOptions private constructor(
     }
 
     /**
-     * Returns a string representation of the object.
+     * Regenerate whenever a change is made
```

Comment:
NIT why did this change?

Reply:
I have no idea, reverted it. The same comments are here on other methods since 7b7d478bf58420d182610bfc7629bd7a26a0de7b